src/compute-client/src/controller/instance.rs

```diff
@@ -1314,7 +1314,8 @@ where
             .into_iter()
             .map(|object| BuildDesc {
                 id: object.id,
-                plan: FlatPlan::from(object.plan),
+                // This should probably be made into a fallible
```

Comment:
That comment looks cut off?

Reply:
Yeah, I changed my mind about the error and must have cmd-z the wrong number of times.